bugsnag/utils.py

```diff
@@ -81,13 +81,15 @@ def default(self, obj):
             bugsnag.logger.exception('Could not add object to payload')
             return self.unencodeable_value
 
-    def _sanitize(self, obj, trim_strings, ignored=None):
+    def _sanitize(self, obj, trim_strings, ignored=None, seen=None):
         """
         Replace recursive values and trim strings longer than
         MAX_STRING_LENGTH
         """
         if not ignored:
             ignored = set()
+        if not seen:
```

Comment:
It seems preferable to compare `seen` to `None` directly, as `not []` is true, so it would create another empty array unnecessarily. However, I can't see how `_sanitize` will be called with an empty `seen` with the current code, so it probably doesn't matter too much.